packages/mongo-nav/src/org-nav/OrgNav.spec.tsx

```diff
@@ -86,6 +87,18 @@ describe('packages/mongo-nav/src/org-nav', () => {
     );
   };
 
+  const testForVersion = (isVisible = true) => {
```

Comment:
To be honest, I think this is fine to just test:
- once, given default props, that this is invisible
- once, given onPrem, that this is visible

I sometimes go overboard in these things, so my apologies if this is different from `testForPaymentStatus` and such, totally my fault.

Reply:
👍